rancid-tomatillos/src/Components/MovieView/MovieView.js

```diff
@@ -48,7 +48,7 @@ class MovieView extends Component{
         return(
             <div className="movieViewError">
             <div style={{display: this.state.error ? "block" : "none"}}> {this.state.error} </div>
-            <div className="movieViewMain" src={film.backdrop_path} style={{ backgroundImage: `url(${film.backdrop_path})` }}>
+                <div className="movieViewMain" src={film.backdrop_path} alt={`scene image from ${film.title}`} style={{ backgroundImage: `url(${film.backdrop_path})` }}>
```

Comment:
Love the creative problem solving here. I'm glad we have at least some information about the images. Maybe we can reach out to the folks who wrote the API and suggest they add some more descriptive alt tags.